### PR TITLE
Frm main climatic edits

### PR DESCRIPTION
**Pull request by shadrackkibet**

### CODE REVIEW

**Comment by shadrackkibet** [comment]
@stevekogo kindly check if this before @dannyparsons  merge the changes

**Comment by stevekogo** [comment]
This addresses what is proposed in #2971 and is ready for @dannyparsons.

File: instat/frmMain.vb [hunk 1]
```diff
@@ -1369,6 +1369,10 @@ Public Class frmMain
         dlgViewGraph.ShowDialog()
     End Sub
 
+    Private Sub mnuClimaticDatesMakeDate_Click(sender As Object, e As EventArgs) Handles mnuClimaticDatesMakeDate.Click
+
+    End Sub
```
Comment (dannyparsons) [inline instat/frmMain.vb]: I think this is a duplicate?

Comment (shadrackkibet) [inline instat/frmMain.vb]: let me check this.

**Comment by shadrackkibet** [comment]
@dannyparsons this is now ready for you.

**Pull request merged**